src/archiefvernietigingscomponent/destruction/tests/test_tasks.py

```diff
@@ -286,6 +288,23 @@ def setUpTestData(cls):
             oas="https://oz.nl/catalogi/api/v1/schema/openapi.json",
         )
 
+    @patch.object(timezone, "now", return_value=datetime.datetime(2021, 11, 1, 00))
+    def time_past(self, m):
+        return timezone.make_aware(timezone.now())
+
+    def test_email_when_delayed(self, m):
+        AutomaticEmailFactory.create(type=EmailTypeChoices.update_required)
+        time_past = self.time_past()
+        destruction_list = DestructionListFactory.create()
+        assignees = DestructionListAssigneeFactory.create_batch(
+            size=2, destruction_list=destruction_list
+        )
+        assignees[0].assigned_on = time_past
+        destruction_list.save()
+        assignees[0].save()
+        send_email_after_time(destruction_list.id)
+        self.assertEqual(len(mail.outbox), 1)
```

Comment:
I would like to see more test cases. Please create a new test class and group all the tests there. Some of the cases that I have in mind are:

- There are 2 lists, each with an assignee, but one assigned since a shorter period than the configured `days_until_reminder` and the other one since a longer period. Only 1 email to be sent.
- A list with an assignee that is not a reviewer and they have been assigned for longer than the `days_until_reminder`. An email will not be sent.
- A single list with two related `DestructionListAssignees` (but only one assigned to the list, and this assignee has been assigned for longer than `days_until_reminder`). This will cause only one email to be sent.